packages/vite-plugin/src/index.ts

```diff
@@ -33,6 +33,16 @@ export function lingui(linguiConfig: LinguiConfigOpts = {}): Plugin[] {
           )
         }
       },
+      resolveDynamicImport(specifier, importer, options) {
+        if (specifier === "@lingui/macro") {
```

Comment:
in v5 it should check for `@lingui/macro` and `@lingui/react/macro` and `@lingui/core/macro`